pkg/sdk/common_table_types.go

```diff
@@ -0,0 +1,72 @@
+package sdk
+
+type RowAccessPolicy struct {
+	With            *bool                  `ddl:"keyword" sql:"WITH"`
+	rowAccessPolicy bool                   `ddl:"static" sql:"ROW ACCESS POLICY"`
+	Name            SchemaObjectIdentifier `ddl:"identifier"`
+	On              []string               `ddl:"keyword,parentheses" sql:"ON"`
+}
+
+// ColumnInlineConstraint is based on https://docs.snowflake.com/en/sql-reference/sql/create-table-constraint#inline-unique-primary-foreign-key.
+type ColumnInlineConstraint struct {
+	NotNull    *bool                 `ddl:"keyword" sql:"NOT NULL"`
+	Name       *string               `ddl:"parameter,no_equals" sql:"CONSTRAINT"`
+	Type       *ColumnConstraintType `ddl:"keyword"`
+	ForeignKey *InlineForeignKey     `ddl:"keyword" sql:"FOREIGN KEY"`
```

Comment:
They share
```
[ [ NOT ] ENFORCED ]
[ [ NOT ] DEFERRABLE ]
[ INITIALLY { DEFERRED | IMMEDIATE } ]
[ ENABLE | DISABLE ]
[ VALIDATE | NOVALIDATE ]
[ RELY | NORELY ]
```
That's why I think @sfc-gh-pbosak combined them. TBH I don't mind having them like this.

Can I make them prettier in the following change? They work now and I would like to avoid another refactoring in this PR.